app/pages/crop-comparison/index.vue

```diff
@@ -33,6 +33,7 @@
               href="https://www.youtube.com/watch?v=nEZzA42Wa2c"
               target="_blank"
               rel="noopener"
+              v-if=false
```

Comment:
Move this to be the first property on the `site-link` component and add double-quotes around `false`